src/components/Checkbox/index.css

```
@@ -0,0 +1,8 @@
+.checkbox {
+    padding: 5px;
+    margin-right: 5px;
+}
+
+.label{
```

```suggestion
label {
```
you're abusing the CSS `class names`